neqo-http3/src/connection_client.rs

```diff
@@ -1248,9 +1245,7 @@ mod tests {
             .fetch(
                 now(),
                 "GET",
-                "https",
-                "something.com",
-                "/",
+                "https://something.com/",
```

Comment:
This is the "easy" form of the interface, but it is the least efficient as the string needs to be parsed into a URL.  This hides that cost somewhat, which is un-rust-like, but ergonomics >> purity.